inventory/def.cf

```diff
@@ -0,0 +1,26 @@
+bundle agent inventory_def_file_control
+{
+  vars:
+  # Each input should be guarded with both an enable and !disable class so that
```

Comment:
I think classes are the right way to signal this. It allows you to collect inventory for a subset of your infrastructure instead of for the whole thing. To implement the same then each host would have to have its own centrally defined list.